src/main/java/org/gridsuite/modification/server/service/NetworkModificationService.java

```diff
@@ -1265,18 +1278,17 @@ private List<EquipmenModificationInfos> execCreateVoltageLevel(
                     throw new NetworkModificationException(CREATE_VOLTAGE_LEVEL_ERROR, "Swich kind '" + switchKind + "' unknown");
                 }
             }
+        }
 
-            subReporter.report(Report.builder()
-                .withKey("voltageLevelCreated")
-                .withDefaultMessage("New voltage level with id=${id} created")
-                .withValue("id", voltageLevelCreationInfos.getEquipmentId())
-                .withSeverity(new TypedValue("VOLTAGE_LEVEL_CREATION_INFO", TypedValue.INFO_LOGLEVEL))
-                .build());
+        subReporter.report(Report.builder()
+            .withKey("voltageLevelCreated")
+            .withDefaultMessage("New voltage level with id=${id} created")
+            .withValue("id", voltageLevelCreationInfos.getEquipmentId())
+            .withSeverity(new TypedValue("VOLTAGE_LEVEL_CREATION_INFO", TypedValue.INFO_LOGLEVEL))
+            .build());
 
-            // add the voltage level creation entity to the listener
-            listener.storeVoltageLevelCreation(voltageLevelCreationInfos);
-        }, CREATE_VOLTAGE_LEVEL_ERROR, networkUuid, reporter, subReporter).stream().map(EquipmenModificationInfos.class::cast)
-            .collect(Collectors.toList());
+        // add the voltage level creation entity to the listener
+        listener.storeVoltageLevelCreation(voltageLevelCreationInfos);
```

Comment:
I think this line should not be called when it is a voltage level creation done in a line split modification, because, in
that case, the voltage level creation entity will be created in the 'storeLineSplitWithVoltageLevelInfos' method of NetworkStoreListener class ...

Reply:
You are right, this what happen currently.